[2025-01-19T19:38:45.567+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-19T19:38:45.579+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.transform_cnaes manual__2025-01-19T17:44:50.357961+00:00 [queued]>
[2025-01-19T19:38:45.583+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: coleta_dados_abertos_cnpj.transform_cnaes manual__2025-01-19T17:44:50.357961+00:00 [queued]>
[2025-01-19T19:38:45.584+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2025-01-19T19:38:45.614+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_cnaes> on 2025-01-19 17:44:50.357961+00:00
[2025-01-19T19:38:45.618+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=85) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-19T19:38:45.619+0000] {standard_task_runner.py:72} INFO - Started process 86 to run task
[2025-01-19T19:38:45.619+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'coleta_dados_abertos_cnpj', 'transform_cnaes', 'manual__2025-01-19T17:44:50.357961+00:00', '--job-id', '208', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados_abertos_cnpj/dag.py', '--cfg-path', '/tmp/tmpu87vzowg']
[2025-01-19T19:38:45.620+0000] {standard_task_runner.py:105} INFO - Job 208: Subtask transform_cnaes
[2025-01-19T19:38:45.657+0000] {task_command.py:467} INFO - Running <TaskInstance: coleta_dados_abertos_cnpj.transform_cnaes manual__2025-01-19T17:44:50.357961+00:00 [running]> on host f9324e49080a
[2025-01-19T19:38:45.701+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Corvaum' AIRFLOW_CTX_DAG_ID='coleta_dados_abertos_cnpj' AIRFLOW_CTX_TASK_ID='transform_cnaes' AIRFLOW_CTX_EXECUTION_DATE='2025-01-19T17:44:50.357961+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-19T17:44:50.357961+00:00'
[2025-01-19T19:38:45.702+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-19T19:38:45.710+0000] {logging_mixin.py:190} INFO - cnaes
[2025-01-19T19:38:45.710+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-19T19:38:45.716+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-19T19:38:45.716+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=coleta_dados_abertos_cnpj, task_id=transform_cnaes, run_id=manual__2025-01-19T17:44:50.357961+00:00, execution_date=20250119T174450, start_date=20250119T193845, end_date=20250119T193845
[2025-01-19T19:38:45.752+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-19T19:38:45.762+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-19T19:38:45.764+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
